config/locales/en.txt

```diff
@@ -1091,7 +1091,7 @@
   log_ancient: "[String]"
   log_approved_by: Approved by [user].
   log_approved_by_with_comment: "[:log_approved_by] [comment]"
-  log_consensus_changed: Consensus changed from [old] to [new].
+  log_consensus_changed: "Consensus changed from: [old]."
```

Comment:
Do you remember why [new] is being removed?

Reply:
That may have been my own flourish, it's starting to ring a bell.

Is it reliable assumption that for _a log of an observation name change_, [new] == the title of the log item? (If a log is generated every time consensus changes...) In that case i felt it was less verbose, but it seems like I would have run that by you at the time 🤔 

Or is that an unreliable assumption, and the title of the log will always be the current (as of right now) consensus — so if the consensus changes three times, it's not accurately showing from what to what each time...?

Reply:
@pellaea This was your thinking on this matter at the time. 

> PS. log_consensus_changed -- This is a tricky one.  I think the motivation was to shorten the matrix box, because the item will already have the latest name, so if you leave it as "Consensus changed from [old] to [new]" the new name will appear twice in the matrix box, and names (esp. with the authorities!) can be huge.  But when you ask to see the log for that observation, it will just show the old name, not the new name.  You see, this override thing was all about allowing us to display something slightly different for the observation / activity index versus the "Show Log" for an observation.  ...   Let's just make them the same, and use the version that looks best on the activity log / observation index page (matrix box).  So that means removing "[new]" from log_concensus_changed.  Make sense?